frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -39,8 +41,25 @@ export interface IPropertyFilterProps {
   sort?: TableSort<any>;
   /** Event fire when sorting changes. */
   onSorting?: (sort: TableSort<any>) => void;
+  /** Show select with my agencies/All Government dropdown */
+  showAllAgencySelect?: boolean;
```

Comment:
required because this filter is reused on the inventory page and we don't want the all properties toggle there.